cpp/src/arrow/compute/kernels/scalar_temporal.cc

```diff
@@ -602,6 +685,15 @@ const FunctionDoc subsecond_doc{
      "Returns an error if timestamp has a defined timezone. Null values return null."),
     {"values"}};
 
+const FunctionDoc strftime_doc{
+    "Convert timestamps to a string representation with an arbitrary format",
+    ("Strftime returns a string representation with an arbitrary format.\n"
+     "Returns an error if timestamp has a defined timezone.\n"
```

Comment:
What does this mean?  I don't see code anywhere that errors if the timestamp has a timezone but I might just not be seeing it.

Reply:
Oh, sorry forgot to delete that.